_data/misc.yaml

```diff
@@ -0,0 +1,45 @@
+releases:
+  core:
+    latest:
+      1-11: 1.11.0-RC1
+      1-10: 1.10.2
+      1-9: 1.9.3
+    all:
+      - "1.1"
+      - "1.2"
+      - "1.3"
+      - "1.4"
+      - "1.5"
+      - "1.6"
+      - "1.7"
+      - "1.8"
+      - "1.9"
+      - "1.10"
+      - "1.11"
+  plugins:
+    foreman_ansible:
+      - "0.x"
```

Comment:
By the way you can get the lists from https://rubygems.org/gems/foreman_discovery/versions